src/config/glean.js

```diff
@@ -42,7 +42,7 @@ export const FIREFOX_ON_GLEAN = {
       values: [
         { key: '*', label: 'All OSes' },
         { key: 'Windows', label: 'Windows' },
-        { key: 'Mac', label: 'Mac' },
+        { key: 'Darwin', label: 'Mac' },
         { key: 'Linux', label: 'Linux' },
```

Comment:
Looking at this [ingestion code](https://github.com/mozilla/gcp-ingestion/blob/a6928fb089f1652856147c4605df715f327edfcd/ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/NormalizeAttributes.java#L52-L74), I think we're missing more than just Mac?

Reply:
Interesting. I did find a very small amount of BSD in the data. The thing is if we want to accommodate more than one name into a single alias, we have to do that at the data level, otherwise the aggregations won't make sense. I say we land this so ppl can look at Mac data and open an issue for your find?

Reply:
I agree. Maybe good to use this [normalized os UDF](https://mozilla.github.io/bigquery-etl/mozfun/norm/#os-udf) at the ETL level.

Reply:
Perfect! Thanks for finding this. Would you mind opening the ticket with this precious info?

Reply:
done!

https://github.com/mozilla/glam/issues/2819